Tests/Validator/UniqueUrlValidatorTest.php

```diff
@@ -99,6 +102,7 @@ public function testValidateWithRootUrlAndNoParent()
 
     public function testValidateWithPageDynamic()
     {
+        $this->skipInPHP55();
```

Comment:
Thanks, but this is probably a different quirk, because I'm not importing `ExecutionContextInterface` anywhere.